src/arrayMethodSort.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
+
+    if (!callBack) {
+      callBack = function (a, b) {
+        return String(a) > String(b) ? 1 : -1;
+      };
+    }
+
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let a = 0; a < this.length - 1 - i; a++) {
+        if (callBack(this[a], this[a + 1]) > 0) {
+          const temp = this[a];
+
+          this[a] = this[a + 1];
+          this[a + 1] = temp;
```

Comment:
You can use array destructuring to swap elements, which can make the code a bit cleaner: `[this[a], this[a + 1]] = [this[a + 1], this[a]]`.